src/app/page.tsx

```diff
@@ -24,11 +26,23 @@ export default function Home() {
     ) : null;
   }
 
+  function getAccountHistory() {
+    //gets Orderlist
+    //TODO:
+    //Rerender this component when wallet is disconnected
+    //Rerender when a transaction has been made
+    const account = wallet ? wallet.accounts[0]?.address : "";
+    return adexState.status != "LOADING" ? (
+      <AccountHistory account={account} />
+    ) : null;
+  }
+
```

Comment:
I don't think we really want to show nothing when we have no data. We've done so with other components in the past, but that's not good - will cause layout shifts and page jumps when stuff loads. Better to show some empty rows.

Reply:
This should still render a the table headers but with no row. Will check again and test to at least have 1 empty row.